README.md

```diff
@@ -23,7 +23,7 @@ var session = require('express-session')
 Create a session middleware with the given `options`.
 
 **Note** Session data is _not_ saved in the cookie itself, just the session ID.
-Session hdata is stored server-side.
```

Comment:
Gotcha darn it, this new keyboard I have loves to sneak stay "h" character everywhere...